README.md

```diff
@@ -108,7 +108,9 @@ Note that you should configure accessing the SCIM bridge credentials through eit
 |-----|------|---------|-------------|
 | name | string | `op-scim-bridge-credentials` | Secret name. |
 | key | string | `scimsession` | Secret key. |
-| value | string | `base64 encoded scimsession file` | Base64 encoded scimsession file. |
+| value_json | string | `{}` | JSON contents of the scimsession file. |
+| value_base64 | string | `""` | Base64 encoded contents of the scimsession file. |
```

Comment:
base64, not url safe base64, correct? I don't *think* there are any components that would require url safe, but this has caught me before.

Reply:
Correct. In my testing `base64` (and `openssl`) worked fine to encode the `scimsession` contents. I think making the distinction in the docs will likely confuse users more than help them.